android/pytorch_android/src/main/java/com/facebook/pytorch/PytorchScriptModule.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.pytorch;
+
+import com.facebook.jni.HybridData;
+
+import java.nio.Buffer;
+
+public class PytorchScriptModule {
```

Comment:
does it need to have Pytorch in the name? it's already in the namespace. Also, in C++ it's called just `Module` - can we use the same naming please?